Description: Clean up spelling errors and other nits in man pages

Author: Theodore Ts'o <tytso@mit.edu>
---
 e2fsck/e2fsck.conf.5.in |  2 +-
 misc/e2image.8.in       |  2 +-
 misc/ext4.5.in          | 30 ++++++------------------------
 misc/mke2fs.conf.5.in   |  2 +-
 misc/tune2fs.8.in       | 32 ++++++++++++++++++++++++--------
 5 files changed, 33 insertions(+), 35 deletions(-)

diff --git a/e2fsck/e2fsck.conf.5.in b/e2fsck/e2fsck.conf.5.in
index 1f80a04..b6fdafe 100644
--- a/e2fsck/e2fsck.conf.5.in
+++ b/e2fsck/e2fsck.conf.5.in
@@ -179,7 +179,7 @@ or
 are not available or are not yet writeable, e2fsck will save the output
 in a memory buffer, and a child process will periodically test to see if
 the log directory has become available after the boot sequence has
-mounted the requiste filesytem for reading/writing.  This implements the
+mounted the requiste file system for reading/writing.  This implements the
 functionality provided by
 .BR logsave (8)
 for e2fsck log files.
diff --git a/misc/e2image.8.in b/misc/e2image.8.in
index 75002d7..f28a76e 100644
--- a/misc/e2image.8.in
+++ b/misc/e2image.8.in
@@ -62,7 +62,7 @@ or
 .B \-Q
 options, the filesystem must be unmounted or be mounted read/only, in order
 for the image file to be in a consistent state.  This requirement can be
-overriden using the
+overridden using the
 .B \-f
 option, but the resulting image file is very likely not going to be useful.
 .PP
diff --git a/misc/ext4.5.in b/misc/ext4.5.in
index e892743..902c332 100644
--- a/misc/ext4.5.in
+++ b/misc/ext4.5.in
@@ -20,7 +20,7 @@ mounted using the ext4 file system driver, and indeed in many modern
 Linux distributions, the ext4 file system driver has been configured
 handle mount requests for ext2 and ext3 file systems.
 .SH FILE SYSTEM FEATURES
-A file system formated for ext2, ext3, or ext4 can be have some
+A file system formatted for ext2, ext3, or ext4 can be have some
 collection of the follow file system feature flags enabled.  Some of
 these features are not supported by all implementations of the ext2,
 ext3, and ext4 file system drivers, depending on Linux kernel version in
@@ -271,7 +271,9 @@ is determined by the filesystem superblock. Set them with
 .BR tune2fs (8).
 .TP
 .BR acl | noacl
-Support POSIX Access Control Lists (or not).
+Support POSIX Access Control Lists (or not).  See the
+.BR acl (5)
+manual page.
 .TP
 .BR bsddf | minixdf
 Set the behavior for the
@@ -382,16 +384,6 @@ The ext3 filesystem is a version of the ext2 filesystem which has been
 enhanced with journaling.  It supports the same options as ext2 as
 well as the following additions:
 .TP
-.B journal=update
-Update the ext3 filesystem's journal to the current format.
-.TP
-.B journal=inum
-When a journal already exists, this option is ignored. Otherwise, it
-specifies the number of the inode which will represent the ext3 filesystem's
-journal file; ext3 will create a new journal, overwriting the old contents
-of the file whose inode number is
-.IR inum .
-.TP
 .BR journal_dev=devnum / journal_path=path
 When the external journal device's major/minor numbers
 have changed, these options allow the user to specify
@@ -456,11 +448,6 @@ Enable Extended User Attributes. See the
 .BR attr (5)
 manual page.
 .TP
-.B acl
-Enable POSIX Access Control Lists. See the
-.BR acl (5)
-manual page.
-.TP
 .BR usrjquota=aquota.user | grpjquota=aquota.group | jqfmt=vfsv0
 Apart from the old quota system (as in ext2, jqfmt=vfsold aka version 1 quota)
 ext3 also supports journaled quotas (version 2 quota). jqfmt=vfsv0
@@ -477,7 +464,7 @@ filesystem.
 The options
 .B journal_dev, norecovery, noload, data, commit, orlov, oldalloc, [no]user_xattr
 .B [no]acl, bsddf, minixdf, debug, errors, data_err, grpid, bsdgroups, nogrpid
-.B sysvgroups, resgid, resuid, sb, quota, noquota, grpquota, usrquota
+.B sysvgroups, resgid, resuid, sb, quota, noquota, nouid32, grpquota, usrquota
 .B usrjquota, grpjquota and jqfmt
 are backwardly compatible with ext3 or ext2.
 .TP
@@ -582,13 +569,8 @@ block device when blocks are freed.  This is useful for SSD devices and
 sparse/thinly-provisioned LUNs, but it is off by default until sufficient
 testing has been done.
 .TP
-.B nouid32
-Disables 32-bit UIDs and GIDs.  This is for
-interoperability  with  older kernels which only
-store and expect 16-bit values.
-.TP
 .BR block_validity / noblock_validity
-This options allows to enables/disables the in-kernel facility for tracking
+This options enables/disables the in-kernel facility for tracking
 filesystem metadata blocks within internal data structures. This allows multi-\c
 block allocator and other routines to quickly locate extents which might
 overlap with filesystem metadata blocks. This option is intended for debugging
diff --git a/misc/mke2fs.conf.5.in b/misc/mke2fs.conf.5.in
index 0830d2e..1ce0f5e 100644
--- a/misc/mke2fs.conf.5.in
+++ b/misc/mke2fs.conf.5.in
@@ -484,7 +484,7 @@ requested alignment.  If this relation is not specified, no alignment
 requirement will be imposed on the huge files.
 .TP
 .I hugefiles_align_disk
-Thie relations specifies whether the alignment should be relative to the
+This relations specifies whether the alignment should be relative to the
 beginning of the hard drive (assuming that the starting offset of the
 partition is available to mke2fs).  The default value is false, which
 will cause hugefile alignment to be relative to the beginning of the
diff --git a/misc/tune2fs.8.in b/misc/tune2fs.8.in
index 15cd967..fc326a1 100644
--- a/misc/tune2fs.8.in
+++ b/misc/tune2fs.8.in
@@ -546,8 +546,15 @@ Use hashed b-trees to speed up lookups for large directories.
 .B dir_nlink
 Allow more than 65000 subdirectories per directory.
 .TP
+.B encrypt
+Enable file system level encryption.
+.B Tune2fs
+currently only supports setting this filesystem feature.
+.TP
 .B extent
 Enable the use of extent trees to store the location of data blocks in inodes.
+.B Tune2fs
+currently only supports setting this filesystem feature.
 .TP
 .B extra_isize
 Enable the extended inode fields used by ext4.
@@ -576,29 +583,37 @@ Support files larger than 2 terabytes in size.
 .B large_file
 Filesystem can contain files that are greater than 2GB.
 .TP
-.B resize_inode
-Reserve space so the block group descriptor table may grow in the
-future.
-.B Tune2fs 
-only supports clearing this filesystem feature.
+.B metadata_csum
+Store a checksum to protect the contents in each metadata block.
 .TP
 .B mmp
 Enable or disable multiple mount protection (MMP) feature.
 .TP
+.B project
+Enable project ID tracking.  This is used for project quota tracking.
+.TP
 .B quota
 Enable internal file system quota inodes.
 .TP
 .B read-only
 Force the kernel to mount the file system read-only.
 .TP
+.B resize_inode
+Reserve space so the block group descriptor table may grow in the
+future.
+.B Tune2fs
+only supports clearing this filesystem feature.
+.TP
 .B sparse_super
 Limit the number of backup superblocks to save space on large filesystems.
+.B Tune2fs
+currently only supports setting this filesystem feature.
 .TP
 .B uninit_bg
 Allow the kernel to initialize bitmaps and inode tables lazily, and to
 keep a high watermark for the unused inodes in a filesystem, to reduce
 .BR e2fsck (8)
-time.  This first e2fsck run after enabling this feature will take the
+time.  The first e2fsck run after enabling this feature will take the
 full time, but subsequent e2fsck runs will take only a fraction of the
 original time, depending on how full the file system is.
 .RE
@@ -610,8 +625,9 @@ After setting or clearing
 or
 .B resize_inode
 filesystem features,
+the file system may require being checked using
 .BR e2fsck (8)
-must be run on the filesystem to return the filesystem to a consistent state.
+to return the filesystem to a consistent state.
 .B Tune2fs
 will print a message requesting that the system administrator run
 .BR e2fsck (8)
@@ -724,7 +740,7 @@ WARNING: The undo file cannot be used to recover from a power or system crash.
 .SH BUGS
 We haven't found any bugs yet.  That doesn't mean there aren't any...
 .SH AUTHOR
-.B tune2fs 
+.B tune2fs
 was written by Remy Card <Remy.Card@linux.org>.  It is currently being
 maintained by Theodore Ts'o <tytso@alum.mit.edu>.
 .B tune2fs
-- 
2.5.0

